contracts/Vault.vy

```diff
@@ -428,6 +432,15 @@ def setRewards(rewards: address):
     self.rewards = rewards
     log UpdateRewards(rewards)
 
+@external
+def setLockedProfitDegration(degration: uint256):
```

Comment:
Should `degration` be limited to within a range?

Reply:
It should not be bigger than the `DEGREDATION_COEFFICIENT`

Reply:
added in 62812d6